huaweicloud/config/config.go

```diff
@@ -15,13 +15,16 @@ import (
 	"github.com/chnsz/golangsdk/openstack/obs"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/mutexkv"
 )
 
 const (
 	obsLogFile         string = "./.obs-sdk.log"
 	obsLogFileSize10MB int64  = 1024 * 1024 * 10
 )
 
+var MutexKV = mutexkv.NewMutexKV()
```

Comment:
seems we already have an `osMutexKV` in provider which is used in `router` and `router_route` resources for the same purpose.

Reply:
already replace them with `config.MutexKV`.